cmake/store_toolchain.cmake

```diff
@@ -3,6 +3,3 @@ set(CMAKE_SYSTEM_VERSION 10.0)
 if (NOT DEFINED CMAKE_SYSTEM_PROCESSOR)
   set(CMAKE_SYSTEM_PROCESSOR ${CMAKE_HOST_SYSTEM_PROCESSOR})
 endif()
-if (NOT onnxruntime_BUILD_FOR_WINDOWS_STORE)
-  set(CMAKE_VS_GLOBALS "AppContainerApplication=false")
```

Comment:
Does VS_GLOBAL_AppContainerApplication VS_GLOBAL_AppContainerApplication  and AppContainerApplication not need to be set to true / false anymore?

Reply:
VS should set it when WINDOWS_STORE, we don't have to change it now